pkg/util/oc/oc.go

```diff
@@ -3,116 +3,151 @@ package oc
 import (
 	"fmt"
 	"math/rand"
-	"strings"
 	"time"
 
-	"github.com/maistra/maistra-test-tool/pkg/util"
 	"github.com/maistra/maistra-test-tool/pkg/util/check/common"
 	"github.com/maistra/maistra-test-tool/pkg/util/shell"
-	"github.com/maistra/maistra-test-tool/pkg/util/template"
 	"github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
 func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+var DefaultOC = NewOC("")
+
+func WithKubeconfig(location string) *OC {
```

Comment:
Only to understand here, if no kubeconfig is given, will take "" as default and will use the default cluster (In the majority of the cases the oc login done before? or we are going to work with kubeconfig file always from now?)

Reply:
You made me remember something about the other `withKubeconfig` function. I'll push another commit shortly.

Reply:
But yes, the KUBECONFIG variable will be set only when we explicitly specify it in `OC`. In other words: most commands will not set this variable.